### ISSUE TITLE
Global options tab should be hidden in Accessibility settings

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6847]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.957
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

Open Help / Accessibility / Accessibility Options...

### Describe the problem in detail

The vertical "Options" Tab containing General, Code , Appearance, etc. is presented, and users can even choose to select non-accessibility-related tabs.

### Describe the behavior you expected

Although "Accessibility" tab is selected by default, screen-reader users can easily get confused by the selectable vertical "Options" tab. Hidding global tabs would be ideal so that this menu only shos accessibility-related settings.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Great feedback, thanks. Yes, right now the Accessibility Options command just brings up the regular Global Options dialog and selects the "Accessibility" tab.

I originally considered making a separate Accessibility Options dialog but went with just adding it to the existing options dialog. I see your point, and it makes sense.

**Comment by kfeinauer** [comment]
Verified fixed in RSP 1.4.940-1 and Desktop 1.4.944
